server/routes/__tests__/money.spec.js

```diff
@@ -339,8 +339,8 @@ describe('Prisoner Money', () => {
           expect(spy).toHaveBeenCalledWith(
             testUser,
             'spends',
-            new Date('2021-01-01T00:00:00.000Z'),
-            new Date('2021-01-31T23:59:59.999Z'),
+            new Date('2021-01-01T00:00:00.000'),
+            new Date('2021-01-31T23:59:59.999'),
```

Comment:
Removing UTC from these tests. The code they work on, only works with local (date_fns). Worth raising a ticket to investigate using the UTC instead.